components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -116,14 +116,9 @@ export default function () {
     }
 
     const formatDate = (date: string | undefined) => {
-        return date ? moment(date).fromNow() : "–";
+        return date ? moment(date).fromNow() : "";
     }
 
-    const formatHash = (longHash: string) => {
-        return longHash ? longHash.substring(0, 8) : "–";
-    }
-
-
     return <>
         <Header title="Prebuilds" subtitle={`View all recent prebuilds for all active branches.`} />
         <div className="lg:px-28 px-10">
```

Comment:
**issue**: I could not see any prebuild to check out the UX on the prebuild page. 😭

Reply:
@gtsiolis please try again!